.webpack/webpack.base.js

```diff
@@ -148,6 +153,11 @@ module.exports = (env, argv, { SRC_DIR, DIST_DIR }) => {
       // Uncomment to generate bundle analyzer
       // new BundleAnalyzerPlugin(),
     ],
+    // enable experimental features, like webAssembly
```

Comment:
Do we really need this? how come our other wasm codecs for loading work without this?